src/sources/readerthread.cpp

```diff
@@ -145,6 +138,10 @@ void ReaderThread::socketNotifierSLOT()
 		m_length = (m_length+1) % FIFO_LENGTH;
 	}
 
+    if (-1 == retval)
```

Comment:
This be written simpler, because only one line the {} are not needed. An simple tab will be ok.

Reply:
Sure, this is a preference thing.... could lead to ambiguous code. I can resubmit?

Reply:
Because on the top of the file you use tabs, (at one line for) So it will be better, when it is uniformly.